### PR TITLE
Requirements - Initial transfer from previous repo

### PR DESCRIPTION
**Pull request by alastc**
Pulling over from the previous Silver repo, this is where all the updates from the [project](https://github.com/orgs/w3c/projects/65/views/1) are included.

You can also [preview via a diff](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2FTR%2Fwcag-3.0-requirements%2F&doc2=https%3A%2F%2Fdeploy-preview-117--wcag3.netlify.app%2Frequirements%2F) (document where the differences are highlighted)

### CODE REVIEW

**Comment by alastc** [comment]
The diff is missing the green ednote styling, but I think line 81 can be removed.

Also, the whole section on issue severity seems to be in the wrong document (starting line 136).

The acknowledgements also seems to be something that can be removed, that is covered in WCAG 3.

File: requirements/index.html [hunk 1]
```diff
@@ -0,0 +1,289 @@
...
+			  <li>Developed problem statements and opportunities to improve accessibility guidance.</li>
+			  <li>Received input from industry leaders for directions to proceed.</li>
+			  <li>Drafted these high-level requirements for the next phase of the project, the prototyping and public input. </li>
+          </ol>
+    </section>
+		<section id="sotd">
+			<p>The Requirements for WCAG 3.0 is published as a joint effort of the <a href="https://www.w3.org/WAI/GL/task-forces/silver/wiki/Main_Page">Silver Task Force</a> of the <a href="https://www.w3.org/WAI/GL/">Accessibility Guidelines Working Group</a> and of the <a href="https://www.w3.org/community/silver/">W3C Silver Community Group</a>. It is a work in progress, and comments are welcome as <a href="https://github.com/w3c/silver/issues">Github Issues</a> or by <a href="mailto:public-silver@w3.org?subject=Comment%20Requirements%20for%20Silver">email</a>. </p>
+			<p>This is the second phase of the Requirements developed during Q1 2019 after months of prototyping work. We anticipate one additional phase in Q3-Q4 2019 after months of evaluating how well actual content can be developed for WCAG 3.0.  </p>
```
Comment (ChrisLoiselle) [inline requirements/index.html:22]: This talks to 2019 and past tense, do we want to update here or are you initially transferring then updating to past tense?

File: requirements/index.html [hunk 2]
```diff
@@ -0,0 +1,289 @@
...
+						<li><strong>digital content</strong>, including guidance currently addressed by WCAG 2.x;</li>
+						<li><strong>authoring tools</strong>, such as content management systems;</li>
+						<li><strong>user agents</strong>, such as browsers and media players;</li>
+						<li><strong>assistive technologies</strong>, such as screen readers, screen magnifiers, and assistants for memory, organization, or simplification;</li>
+						<li><strong>software and web applications</strong>, including mobile apps; and</li>
+						<li><strong>operating systems</strong> and other platforms who may want advice for features to better support people with disabilities.</li>
+					</ul>
+					</li>
+				</ul>
+				<p>The current project design does not intend to write separate specifications or normative requirements for the technology stack beyond digital content. The goal is to provide information that technology venders can choose to use to improve the accessibility of their products to support the guidelines. </p>
```
Comment (ChrisLoiselle) [inline requirements/index.html:77]: venders should be "vendors".

File: requirements/index.html [hunk 3]
```diff
@@ -0,0 +1,289 @@
...
+			<li>Be written so the Guideline content is usable in adaptable and customizable ways. For example, WCAG 3.0 content is available to be extracted by users to adapt to their needs. </li>			
+          </ol>
+</section>
+		<section>
+		  <h2 id="requirements">Requirements</h2>
+          <p>Previous W3C Accessibility Guidelines described how to make web    pages accessible to people with disabilities. These guidelines provided a     flexible framework that has kept the guidelines relevant for   10 years. Changing technology and changing needs of people with   disabilities   has shown areas where they could be improved. The   requirements are   drawn from the research performed by WCAG 3.0 to   improve the guidelines,   and the suggestions from the Silver Design  Sprint. </p>
```
Comment (ChrisLoiselle) [inline requirements/index.html:204]: There is an extra space between for and 10 years. There are also extra spaces before The and requirements, are and drawn and guidelines, and.

File: requirements/index.html [hunk 4]
```diff
@@ -0,0 +1,289 @@
...
+			<h3>Self validation</h3>
+			<p>The WCAG 3 guidelines document shall meet its own requirements where relevant. (Some requirements may apply to organisations or scenarios not relevent to the development of a standard.)</p>
+		</section>
+	</section>
+		<section class="appendix" id="changelog">
+	    <h2>Change Log</h2>
+          <section>
+            <h3 id="changes-before-the-first-public-working-draft">Changes Prior to First Public Working Draft</h3>
+            <ol>
+			  <li>Moved the Opportunity for Evolving Technologies to Maintenance; added new opportunity       for Flexibility in the Conformance section; and  minor grammar corrections. (9 July 2018) </li>
```
Comment (ChrisLoiselle) [inline requirements/index.html:254]: extra space between opportunity and for

File: requirements/index.html [hunk 5]
```diff
@@ -0,0 +1,289 @@
...
+              <li>Add explanatory sentence to Problem Statement intro paragraph   about positive feedback on WCAG guidance and emphasis on structural and   presentation improvements.</li>
+              <li>Fix typo, change "situational disabilities" to "situational limitations"</li>
+              <li>Add short sentences on the time line for updating the Requirements     document based on meeting of Cooper, Lauriat and Spellman of 23 March     2018. </li>
+              <li>Reorder the sections so that the background is first, followed by Design Principles, then Requirements</li>
+              <li>Clarify the broader definition of "disabilities" based on input     from public comments and group discussion. The definition sentence in     the Introduction paragraph is a proposal for group discussion. </li>
+              <li>Change "future" to "unanticipated" in Requirements #2 and add subsequent sentence. </li>
+              <li>Change the Requirements Intro paragraph to remove the specific     sentence on conformance and pass/fail tests and give greater context on     the source of the Requirements.        </li>
```
Comment (ChrisLoiselle) [inline requirements/index.html:268]: extra space between specific and sentence

File: requirements/index.html [hunk 6]
```diff
@@ -0,0 +1,289 @@
...
+              <li>Added headings to individual requirements (25 June 2018)</li>
+              <li>Added section on Comparison to WCAG 2.x (25 June 2018)</li>
+              <li>Changed Problem Statements to Opportunities based on AGWG feedback and group discussion (25 June 2018)</li>
+              <li>Moved Design Principle #10 to #6. (25 June 2018)</li>
+              <li>Changed wording of Multiple Ways to Measure (25 June 2018)</li>
+              <li>Added editor notes for Technology Neutral and Testability (25 June 2018)</li>
+              <li>Added statement to Problem Statements about ATAG &amp; UAAG research. Minor grammar fixes. (3 June 2018)</li>
+              <li>Add explanatory sentence to Problem Statement intro paragraph   about positive feedback on WCAG guidance and emphasis on structural and   presentation improvements.</li>
+              <li>Fix typo, change "situational disabilities" to "situational limitations"</li>
+              <li>Add short sentences on the time line for updating the Requirements     document based on meeting of Cooper, Lauriat and Spellman of 23 March     2018. </li>
```
Comment (ChrisLoiselle) [inline requirements/index.html:264]: Extra space between requirements and document.

File: requirements/index.html [hunk 7]
```diff
@@ -0,0 +1,289 @@
+<!DOCTYPE html>
```
Comment (ChrisLoiselle) [inline requirements/index.html:1]: Extra spaces appear in the green diff version reviewed, recommendation is to remove extra spaces throughout the document, where applicable.

Comment (alastc) [reply requirements/index.html:1]: Done.

File: requirements/index.html [hunk 8]
```diff
@@ -0,0 +1,248 @@
...
+            <li>Be accessible and conform to the Guidelines.</li>
+            <li>Be written in plain language, as easy as possible to understand.</li>
+			<li>Improve the ability to support automated testing where appropriate and provide a procedure for repeatable tests when manual testing is appropriate.</li>
+			<li>Provide requirements to address functional needs, while avoiding or minimizing negative impacts on other functional needs. </li>
+			<li>Include accompanying documents that provide clear, concise, and findable technical information quickly in one place.</li>
+			<li>Include, where possible and appropriate, links to instruction videos, illustrations, and how-to guides. Creation of new videos and illustrations are outside the scope of the working group at this time.</li>
+			<li>Normative requirements should be unambiguous so there is only one possible meaning for each requirement.</li>
+          </ol>
+          <p>The creation process for the guidelines should:</p>
+          <ol start="7">
```
Comment (brianelton) [inline requirements/index.html:154]: This needs to be updated to start="11" as there were 4 items added to the list before it.

**Comment by julierawe** [comment]
FYI, the deploy preview is broken link/"Page not found": https://deploy-preview-117--wcag3.netlify.app/

**Comment by GreggVan** [comment]
1.   a simple ordered list numbering error  (Just above the REQUIRMENTS section) 
2.  in the Changes made prior to first public release  listing 
     - there is a edit that looks like a mistake -- it changed   "**paragraph**"  into  "**paragrap**"    in item  number 9
3.  and did you mean to delete all acknowledgments?

**Comment by alastc** [comment]
> FYI, the deploy preview is broken link/"Page not found": https://deploy-preview-117--wcag3.netlify.app/

Unfortunately the deploy doesn't work for this document, but I put a link in the description at the top.

@brianelton  - thanks, got it.

@GreggVan - Updated, thank you. The acknowledgements are showing as deleted because the preview doesn't show 'includes', but they have not actually been changed.

**Comment by julierawe** [comment]
Re the new "Self validation" section:
- Add hyphen to "Self-validation"?
- Change "organisations" to "organizations" (per WAI style guide)
- Change "relevent" to "relevant"

**Comment by julierawe** [comment]
I appreciate the changes to the Readability section, but we could make the new wording even clearer:
- **October 10th rewording:** Requirements in WCAG 3 for plain language shall be applied to WCAG 3. It is desirable for the guidelines to be understandable by the widest possible audience. 
- **Suggested additional rewording for clarity and ease of reading:** WCAG 3 should be written in plain language. The goal is to make the guidance easy to understand for the widest possible audience.

We could make the wording this simple here because there is a separate section on self-validation that says WCAG 3 needs to meet its own requirements (including for plain language).

**Comment by rachaelbradley** [comment]
@ Julie Suggested rewording: WCAG 3 should be written in plain language. The goal is to make the guidance easy to understand for the widest possible audience.

**Comment by lseeman** [comment]
Hi, Firstly This is a good document. Clearly there is a ton of work in making it etc. A couple of issues follow
1. Are we including mental health related disabilities?
2. Is there a mention of **equity** across disabilities that would be reflected in scoring? In other words. if it is ok to optimize to a disability group, then it can be any disability group, and higher score are not biased towards one group over another but rather reflect need and inclusion. 
3. A commitment  to include a  broad scope of disabilities does not reflect a commitment to include all disabilities. 
4. For Regulatory Environment, I think this should not come at the cost of inclusion. Also the context is so important and is not often well done by regulators . SO for example, all people that can have access to critical information should have that information. If a small petza shop is not fully accessible it is not the same thing at all. We often miss out on giving the advice needed for critical services because it is too much for  all sites such as small websites or user generated content etc. In trying to write legislation we often excluded user groups.  I ams sure there is a solution - in content useable we had a guided to using it in policy. but things should not be exucled  so that policy does not have to think how to included or refence our work.

**Comment by alastc** [comment]
> We could make the wording this simple here because there is a separate section on self-validation that says WCAG 3 needs to meet its own requirements (including for plain language).

The requirements section becomes a measure of the publication, so needs to use testable language. I.e. publication can be blocked if we cannot show it has met the requirement.

Rachael's update works, but if people are happy that the requirement is included in the self-validation requirement, then we should remove the readability requirement.

We have already mentioned it in the Design Principles section, "Be written in plain language, as easy as possible to understand."

**Comment by bruce-usab** [comment]
Commenting here per [requirement review email](https://lists.w3.org/Archives/Public/w3c-wai-gl/2024OctDec/0011.html) to list.

The use of 2.0 / 2.1 / 2.x  seems inconsistent to me.  I am having to judge from the DIFF as I am not finding the deploy preview.

Does the document use "shall" elsewhere?
> WCAG 3 **shall be** written using the plain language guidance from WCAG 3. The goal is for the widest possible audience to understand WCAG 3.

Why not?
> WCAG 3 is written using the plain language guidance from WCAG 3.

**Comment by alastc** [comment]
Hi @lseeman,

Thank you for reviewing this.
 
This review cycle is only on the changes. I think all of these are new issues rather than on the changes made to the document in scope of this PR. Feel free to open new issues if the following doesn’t answer / resolve the question. We will continue refining the requirements document based on issues raised.
 
> 1. Are we including mental health related disabilities?
 
I think we should be guided by how regulators and standards bodies are defining disability. If including requirements for something that wouldn’t be included in regulation would impact uptake, but I’m not sure if that is an issue.
 
In the UK the Equality Act defines it as “a substantial and long-term adverse effect on your ability to carry out normal day-to-day activities”  which covers a lot of mental health conditions.
 
> 2. Is there a mention of equity across disabilities that would be reflected in scoring? In other words. if it is ok to optimize to a disability group, then it can be any disability group, and higher score are not biased towards one group over another but rather reflect need and inclusion.
 
We have a requirement for “Broad disability support”, but nothing specific to the scoring. We haven’t decided to use scoring, that is coming from the requirement to motivate organisations to go beyond minimal requirements.
 
 
> 3. A commitment to include a broad scope of disabilities does not reflect a commitment to include all disabilities.
 
I agree, it does not. Saying “all disabilities” is the sort of thing that gets pushed to absurd places. I would not want our publication to be blocked because there are no provisions for “Ribose-5-Phosphate Isomerase (RPI) deficiency”. (From googling “rare disabilities”, and picking one which probably has no new requirements compared to what is already covered.)
 
> 4. For Regulatory Environment, I think this should not come at the cost of inclusion. Also the context is so important and is not often well done by regulators . SO for example, all people that can have access to critical information should have that information. If a small petza shop is not fully accessible it is not the same thing at all. We often miss out on giving the advice needed for critical services because it is too much for all sites such as small websites or user generated content etc. In trying to write legislation we often excluded user groups. I ams sure there is a solution - in content useable we had a guided to using it in policy. but things should not be exucled so that policy does not have to think how to included or refence our work.
 
Universally applicable guidelines are also not a good place to incorporate context, and there is no way to define “critical information” across all contexts.
What we can do is provide hooks for regulators so they can apply things differently for different contexts. For example, providing requirements tagged with “education” that are most relevant to educational establishments. We can also use “assertions”, which might not be required for conformance to WCAG, but could be required by regulators.
 
Overall, we’re spending quite a bit of time trying to work out how to provide a conformance model and types of provision that cover a lot more than WCAG 2. However, to make a difference it would need to be taken up, so we do need to maintain a good core of foundational requirements.

**Comment by julierawe** [comment]
One suggestion in response to @lseeman comment:

**Add "mental health" to the last sentence in the "Broad disability support" paragraph.**
This sentence already includes new wording—we'd be changing the changes—so this should be in the scope of this pull request: 
"This approach includes particular attention to people whose needs may better be met with a broad testing strategy, such as people with low vision, limited vision, [DELETE: or] cognitive and learning disabilities[ADD: , or mental health disabilities].

**Comment by ashleemboyer** [comment]
Hello! 👋🏻 I've been reading around this repo for a couple of days (mostly the Discussions tab). I hope it's OK to chime in here! If there will be an opening for public comment, happy to hold on to feedback until then too. I'm just really excited about WCAG 3! 🥳

I've used blockquotes to reference specific words and sentences from the page. I have more thoughts than this, but trying to stick to the generated diff. Thanks for your time!

---

> There are needs of people with disabilities, especially cognitive and low vision disabilities, that are better captured by a different type of measurement.

An area where I’ve found WCAG guidelines significantly lacking is around people who have dynamic, multiple, or even conflicting access needs.

I say this as a person with physical, visual, auditory, cognitive, and neurological disabilities and as an accessibility practitioner. The outcome of this, in my experience, is that most people have a narrow, rigid, and biased perspective of what disability and accessibility are.

For example: believing screen readers are only used by people with vision-related disabilities. Another example: not knowing that flashing and layout shift can harm people with vestibular disabilities.

I think it would be valuable to include mention of this here alongside the mentions of cognitive and low vision disabilities, or in the “Broad disability support” section.

> Provide requirements to address functional needs, while avoiding or minimizing negative impacts on other functional needs.

What is the language “functional needs” in reference to?

> Broad disability support

What’s the origin of this language? Is it new?

> Multiple ways to display
> …
> 
> Readability
> …

Both of these section mention different/wide audience, but it’s not clear to me what that means from this page. Are disabled people included this audience, in a variety of contexts (self advocacy, disabled practitioners of accessibility, etc.)?

**Comment by alastc** [comment]
> An area where I’ve found WCAG guidelines significantly lacking is around people who have dynamic, multiple, or even conflicting access needs.

This is a tangent to the topic, but I'm interested in where you might have experience of issues which only happen because of multiple disabilities? I.e. is there a requirement missing from WCAG which is aimed at people with "intersectional needs"? 

For example, I'm very aware that some people use screenreaders (or other text-to-speech) due to cognitive impairments, and obviously there are people with cognitive impairments who are blind or low-vision.  But what issue would only appear because of that combination? 

In usability testing I've seen issues arise that fit reasonably easily into the screenreader-bucket, or the cognitive bucket, but nothing I can recall that only appeared due to the intersection.

> What’s the origin of this [Broad disability support] language? Is it new?

It was changed from "Multiple ways to measure", as the underlying requirement was to incorporate more guidance for more disabilities, and "multiple ways to measure" is the way to achieve it.

> What is the language “functional needs” in reference to?

Functional needs is a way to say: The requirements people with a particular disability have. In standards such as (EN 301-549](https://www.etsi.org/deliver/etsi_en/301500_301599/301549/03.02.01_60/en_301549v030201p.pdf), they call them "Functional performance statements", meaning things like "Usage without vision". They map those to WCAG in Annex B from the link above.

> Both of these section mention different/wide audience, but it’s not clear to me what that means from this page. Are disabled people included this audience, in a variety of contexts (self advocacy, disabled practitioners of accessibility, etc.)?

In the context of a standard aimed at people creating or testing web-technology, yes. So it is aimed at people creating websites (including those with disabilities), but not people consuming those websites (with or without disabilities). Not to say those people couldn't read it, and it would be great if it were understandable, but as a standard the primary audience is the creators and testers (which is an umbrella term that covers about 35 roles).

**Comment by ashleemboyer** [comment]
@alastc 

> This is a tangent to the topic, but I'm interested in where you might have experience of issues which only happen because of multiple disabilities? I.e. is there a requirement missing from WCAG which is aimed at people with "intersectional needs"?

Speaking from my own lived experience -- Color contrast and text customization are great examples. WCAG 2.x has minimum contrast requirements, but no maximum. The minimum is helpful for me because of a few factors of my poor vision, but there are multiple situations that can arise throughout my day when too much contrast causes me different types of pain.

When I have a migraine, for example, I can’t use/read content that’s high contrast and it also makes the migraine worse. I get migraines because of multiple disabilities (i.e. I have several triggers across the sensory spectrum), so it’s not specific to one disability. Nor is it predictable.

In many of these situations, what would enable a path to access are in-site/app settings that I can use to adjust contrast, font size, font weight, etc. I say within the site/app, as opposed to device or browser settings, because notions from 2.x in the tune of “at least one mechanism” leads to products that don’t have *any* built-in mechanisms in practice, because the creators expect devices, browsers, and assistive technology to handle customized experiences.

And the reason why that’s a nice thought in theory, but not in practice, is because it doesn’t work for people who switch between modalities as their access needs change throughout a day. Just speaking in terms of viewport size, I switch between my phone, tablet, laptop, and desktop multiple times a day, based on how my access needs change. Depending on the device, trying to adjust contrast and/or text is tends to be more taxing on mobile devices. For example, there’s no keyboard that enables using CTRL/CMD and the +/- keys to change page-level zoom.

Related, I hypothesize changing built-in presentation settings requires less energy/effort than changing device/browser settings. If that is true (and it wouldn’t have to be true for every person), then the impact of encountering whatever is inaccessible in the moment is reduced, and so is the likelihood that there will be rippling, compounding effects after.

> For example, I'm very aware that some people use screen readers (or other text-to-speech) due to cognitive impairments, and obviously there are people with cognitive impairments who are blind or low-vision. But what issue would only appear because of that combination?

To reiterate/relate some of what I’ve written earlier in this comment (I don’t want to leave this question implicitly answered), it’s not about specific disabilities, but about the access needs that arise in a moment of time as a result of many combinations of factors. It’s really not possible to enumerate, nor has it shown to be very useful to attempt.

To cite some research: a scoping review by Vidya Sundar identified 433 different types of workplace accommodations in the review of 47 studies (2017, 135). Sundar consulted existing research literature to create a clearer picture of how workplace accommodations are used in practice and identify knowledge gaps in how they’re conceptualized. They found several gaps that suggest systematic exclusion of groups of people and types of support, and concludes the need for “further examination of our perception and understanding of disability itself” (152).

> Functional needs is a way to say: The requirements people with a particular disability have. In standards such as EN 301-549, they call them "Functional performance statements", meaning things like "Usage without vision". They map those to WCAG in Annex B from the link above.

Thank you for this context and links! I’ll be doing some reading to better understand use of the term and its intentions.

> In the context of a standard aimed at people creating or testing web-technology, yes. So it is aimed at people creating websites (including those with disabilities), but not people consuming those websites (with or without disabilities). Not to say those people couldn't read it, and it would be great if it were understandable, but as a standard the primary audience is the creators and testers (which is an umbrella term that covers about 35 roles).

I took some time to re-read this doc, and realized where my lack of clarity stems from. In the “Usability” section under “Opportunities for WCAG 3.0” there’s an “Advocacy Tool” paragraph. I traced this back to the “Problem Statements” page in the Silver wiki, where I found a few mentions of advocacy, advocates, lawyer, policy makers in particular. I read some of the research at the bottom of the page and also looked at the Requirements for WCAG 2.0 doc (linked in the “Comparison to WCAG 2.x Requirements” section above).

The “Write to a more diverse audience” requirement lists more than creators and testers as examples. I think changing “a non-technical audience” to “the widest possible audience” is a step backwards to where we are now. I don’t think “non-technical audience” is right answer either, but it’s a step forward. In short, I think it’s important to have an explicit list of audience beyond creators and testers because that list can be a great tool for making sure those kinds of audience members are included throughout the research, design, development, and testing of WCAG 3.

---

Sundar, Vidya. 2017. “Operationalizing Workplace Accommodations for Individuals with Disabilities: A Scoping Review.” Work 6 (1): 135–55. https://doi.org/10.3233/WOR-162472.

**Comment by alastc** [comment]
Hi @ashleemboyer, as this is heading off-topic I've put my response in a [discussion](https://github.com/w3c/wcag3/discussions/123).

**Comment by ashleemboyer** [comment]
> Hi @ashleemboyer, as this is heading off-topic I've put my response in a [discussion](https://github.com/w3c/wcag3/discussions/123).

Great idea! The answer to your tangential question is quite nuanced and multi-layered. It'll be great to have more space to dig in.

---

I have one question about the latest updates (https://github.com/w3c/wcag3/pull/117/commits/8e8d19ac0874cfb3f405ae1c10a146b77715cf76): What definition is being used for the new "intersectional disabilities" terminology?

I looked at the minutes for the 22 October AGWG Teleconference (https://www.w3.org/2024/10/22-ag-minutes) and I see your use of "intersectional needs" a few replies up from here (https://github.com/w3c/wcag3/pull/117#issuecomment-2429494334). Since you added quotation marks around "intersectional needs," I'd like to make sure it's clear that is not language I've suggested. The language I used is "dynamic, multiple, or even conflicting access needs."

I understand what intersectional _needs_ are (race, class, gender, etc. are taken into account in addition to disability), but it's unclear to me what's meant by intersectional _disabilities_.

**Comment by alastc** [comment]
Approved: https://lists.w3.org/Archives/Public/w3c-wai-gl/2024OctDec/0048.html

**Review by iadawn — APPROVED** [review]

**Pull request merged**